### ISSUE TITLE
Quick review of Get There ATX Website fo a11y

### ISSUE THREAD

**Issue by mateoclarke** [cityofaustin/atd-data-tech#399]
links to website and google doc are in the email

**Comment by mateoclarke** [comment]
@amenity AG gave us until Wed. Do you think you'll have time before then to add input. She also invited us to their meeting with the web design consultant. I'm not planning to go, but up to you if you have bandwidth. Only think its important if we find some showstopper issue.

**Comment by amenity** [comment]
Email to Ashley: 

> Hi Ashley, 
> 
> The site looks really nice, great work!
> 
> There are a few accessibility issues that the vendor should be able to address very easily:
> 
> - **Provide meaningful alternative text for images** — For example, the alternative text provided for the [site logo/wordmark](http://gtatx.wpengine.com/wp-content/uploads/2019/01/gtatx-logo-concept-01-600w.png) is just "Image." So someone who is vision impaired or using a language translation tool won't be able to comprehend the header of the website. 
> - **Remove empty links** — For example, the social media icons will be invisible to screenreaders. 
> - **Ensure the document language is set** so that screen readers will read the language correctly. 
> 
> There are other issues you can [check out using the WAVE website](http://wave.webaim.org/report#/http://gtatx.wpengine.com/), but those are the ones that stand out to me. 
> 
> I definitely recommend using WAVE when reviewing websites someone is building for you. You can also include language in vendor contracts that stipulate this requirement. Here's a [random example](https://www.colorado.edu/ictintegrity/required-contract-language-provisions) that Google turned up.
> 
> 
> Cheers,
> 
> Amenity
